### PR TITLE
Let's log a critical error with rollbar

### PR DESCRIPTION
**Pull request by ocke**
:v:

/cc @zendesk/vegemite

### Description
In the aftermath of the S3 cache. Let's log a critical error if the app isn't loaded.

### References
* JIRA: 

### Risks

### CODE REVIEW

File: lib/zendesk_apps_support/assets/installed.js.erb [hunk 1]
```diff
@@ -6,6 +6,8 @@
   <% installations.each do |installation| %>
     if (ZendeskApps[<%= installation.app_name.to_json %>]) {
       ZendeskApps[<%= installation.app_name.to_json %>].install(<%= installation.to_json %>);
+    } else {
+      Rollbar.critical("App code missing: '<%= installation.app_name.to_json %>' (<%= installation.app_id.to_json %>)");
```
Comment (sandlerr) [inline lib/zendesk_apps_support/assets/installed.js.erb]: won't this just error out in Chat? not sure we have rollbar there.

Comment (fvzzy) [inline lib/zendesk_apps_support/assets/installed.js.erb]: I thought we (also) wanted this error going to Datadog?

Comment (ocke) [inline lib/zendesk_apps_support/assets/installed.js.erb]: oh ok, so `Rollbar && `?

Comment (ocke) [inline lib/zendesk_apps_support/assets/installed.js.erb]: btw, i found the errors. they show up in `lotus`

**Comment by sandlerr** [comment]
we need to make the errors should show up in zaf, not in lotus.

**Pull request closed without merging**